### ISSUE TITLE
Submitting ideas through an API call would be cool.

### ISSUE THREAD

**Issue by ocdtrekkie** [simonv3/IdeaOtter#9]
I kinda picture this thing where I tell my home automation system to save an idea, and rather than storing it in it's own table somewhere, it sends it to my Idea Otter grain on Sandstorm. Not sure how much actual value there is in this request, so I'm calling it a "would be cool".

**Comment by simonv3** [comment]
Yeah, I definitely have to inspect how Sandstorm does APIs - there's a couple of things I've been thinking of that would be really useful like this.
